staticx/hooks/glibc.py

```diff
@@ -13,6 +14,8 @@ def process_glibc_prog(sx):
     try:
         nssfix = copy_asset_to_tempfile(LIBNSSFIX, debug=sx.debug,
                 prefix='libnssfix-', suffix='.so')
+        umask=os.umask(0o755)
```

Comment:
FYI: This is bad because it *changes the umask* of the currently-running process to `0o755`.